src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Resources/MarkAssembliesWithNeutralResourcesLanguage.cs

```diff
@@ -53,9 +53,26 @@ public override void Initialize(AnalysisContext context)
 
             context.RegisterCompilationStartAction(cc =>
```

Comment:
As you have done some extra refactorings, it might be worth also renaming `cc` into `context`.